doc/source/releases/15.10_announce.rst

```diff
@@ -1,9 +1,81 @@
+
 ===========================================================
 October 2015 Galaxy Release (v 15.10)
 ===========================================================
 
+.. include:: _header.rst
+
+Highlights
+===========================================================
+
+**Reports Application**
+  The reports web application has been greatly enhanced - including
+  improved styling, new landing page, more reports, pagination, and
+  sparklines. Huge thanks to Daniel Bouchard
+  (`@Airistotal <https://github.com/Airistotal>`__) for these
+  enhancements.
+
+**Upload**
+  The Galaxy upload widget now features support for composite datatypes and
+  improved styling.
+
+**Data Libraries**
+  Improved API and UI for data libraries - including support for library 
+  folder management and search.
+
+`Github <https://github.com/galaxyproject/galaxy>`__
+===========================================================
+
+New
+  .. code-block:: shell
+
+      % git clone -b master https://github.com/galaxyproject/galaxy.git
+
+Update to latest stable release
+  .. code-block:: shell
+
+      % git checkout master && pull --ff-only origin master
+
+Update to exact version
+  .. code-block:: shell
+
+      % git checkout v15.10
+
+
+`BitBucket <https://bitbucket.org/galaxy/galaxy-dist>`__
+===========================================================
+
+Upgrade
+  .. code-block:: shell
+
+      % hg pull
+      % hg update latest_15.10
 
-Schedule
+
+See `our wiki <https://wiki.galaxyproject.org/Develop/SourceCode>`__ for additional details regarding the source code locations.
+
+
+Deprecation Notices
 ===========================================================
- * Planned Freeze Date: October 5, 2015
- * Planned Release Date: October 26, 2015
+
+The Mercurial repository at https://bitbucket.org/galaxy/galaxy-dist is deprecated.
+**We recommend deployers to switch their instances to git** and follow the `master`
+branch of GitHub repository https://github.com/galaxyproject/galaxy
+Details are available at https://wiki.galaxyproject.org/Develop/SourceCode
```

Comment:
We also have https://wiki.galaxyproject.org/Admin/SwitchingToGithubFromBitbucket , can we merge the two pages?